ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ProfileSelection.tsx

```diff
@@ -142,11 +190,13 @@ function ProfileSelection({ profiles, isFetchingProfiles }: ProfileSelectionProp
                                     isSelected: allRowsSelected,
                                 }}
                             />
-                            <Th>Name</Th>
-                            <Th>Description</Th>
+                            <Th />
+                            <Th>Profile</Th>
+                            <Th>Rule set</Th>
```

Comment:
I noticed that the version isn't included in this list, but we appear to have it available to us in the response. Do we want to include it?

```
$ ./bin/roxcurl "/v2/compliance/profiles/cluster?clusterId=c9f7c2ca-f8c5-4017-8064-9e2a4ee5ed77" | jq ".profiles[3].profileVersion"
"Revision 4"
```

Reply:
^ @alanonthegit @rhdcaspin 

What do y'all think of Lance's suggestion?

Reply:
Yeah I'd probably defer to them as well^. I noticed that in the response but wasn't sure how useful that info was.

fyi to @alanonthegit @rhdcaspin it would take <10min to add that column

Reply:
@rhmdnd @vjwilson @bradr5 Ah yes, I think that's a good addition to have. I think profile versioning is something we'll eventually have to dig into further with tailored profiles, but displaying it now doesn't hurt and it could be informative to a user.